TLM/TLM/Manager/Impl/SpeedLimitManager.cs

```diff
@@ -816,28 +795,26 @@ public class SpeedLimitManager : AbstractGeometryObservingManager, ICustomDataMa
 		}
 
 		public bool LoadData(Dictionary<string, float> data) {
-			bool success = true;
+			const bool success = true;
```

Comment:
What's the purpose of this `const`? Isn't the point of having a `success` variable to set it to _false_ if it fails? How will we set it if it's become a constant?

Reply:
My IDE said this variable doesn't change, and suggested const. I was happy to agree. So someone else created it, forgot to change it and left it as `true`. Brought your attention to it, so the change was worth it.

Reply:
If it doesn't change, why keep it? Supposedly this variable is supposed to let us know if it was a successful operation or not, and in this situation it would always be true. No point in setting a useless variable.

Reply:
Removed